e2e/test/scenarios/collections/instance-analytics.cy.spec.js

```diff
@@ -142,5 +144,86 @@ describeEE("scenarios > Instance Analytics Collection", () => {
         cy.contains("Move").should("not.exist");
       });
     });
+
+    it.skip("should not allow editing analytics content (#36228)", () => {
+      // dashboard
+      navigationSidebar().findByText(ANALYTICS_COLLECTION_NAME).click();
+      cy.findByTestId("pinned-items")
+        .findByText(METRICS_DASHBOARD_NAME)
+        .scrollIntoView()
+        .click();
+
+      cy.wait("@getDashboard");
+
+      cy.findByTestId("dashboard-header").within(() => {
+        cy.icon("pencil").should("not.exist");
+      });
+
+      // model
+      navigationSidebar().findByText(ANALYTICS_COLLECTION_NAME).click();
+      cy.findByTestId("pinned-items")
+        .findByText(PEOPLE_MODEL_NAME)
+        .scrollIntoView()
+        .click();
+
+      cy.wait("@getCard");
+
+      cy.findByTestId("qb-header").within(() => {
+        cy.icon("ellipsis").click();
+      });
+
+      popover().findByText("Edit query definition").should("not.exist");
+    });
+  });
+
+  describe("API tests", () => {
```

Comment:
@nemanjaglumac would love to hear your thoughts on this. This is a little experimental.

- we make a series of API requests to check the `can_write` property on every single item in the analytics collection to ensure that it's set to `false`.
- since there's no UI involved, this test actually runs in a fraction of the time it would take to check a single item through the UI, and we get coverage on everything.
- above, we also do a standard UI test on one model and one dashboard.

WDYT? is this valuable? is this a good pattern?